data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+
+
+
+mission "Unfettered: I Shall Go Unpunished"
+	landing
+	source
+		government "Pirate"
+	to offer
+		random < 40
+		has "Unfettered: Jump Drive Retrieval: done"
+		has "flagship crew" > 4
+	on offer
+		require "Gold Bars" 5
+		conversation
+			`As you land on the pirate world, you head for the hatch, preparing to get on the surface, but passing by the cargo bay you think you hear a sound. Odd... There should not be anyone there right now, knowing your crew's habits.`
+			choice
+				`(Its nothing, probably just a rat).`
+					goto lostbars
+				`(Go investigate).`
+			`Arming yourself of your side weapon, you approach the bay as discretely as you can. Worst case scenario is you find one of your own crew, in which case you better not overeact, or someone could get hurt.`
+			choice
+				`(Ask if someone is there).`
+					goto hello
+				`(Never too careful).`
+			`You go into the cargo bay, on your guards. As you enter you think you saw something move, on your left. There is not much time to think.`
+			choice
+				`(Call my crew to help).`
+				`(Run in that direction).`
+					goto run
+			label help
+			`You call your crew to help and try to barricade the ship to prevent escape, but they cannot find anyone. You wonder how they could get away. Just to be sure, you check around the cargo bay. It seems like nothing is missing, but a few bars of golds are out of place. Maybe it was just a malfunction with the ship's gravitational systems.`
+				defer
+			label run
+			`As you run you see a silouette running too. It is armed and you think you recognize the uniform of a member of your crew. Could he be running after the intruder too? Is this just a big misunderstanding?`
+			choice
+				`(Order him to stand down).`
+				`(Follow him to see where he is going).`
+					goto follow
+				`(Shoot).`
+					goto shoot
+			label shout
+			`There is no doubt that you were heard, but the only reaction from the silouette is to run faster, and by another, more direct path. You cannot catch them and they evade your ship. Thankfully they did not manage to steal anything, only to move a few gold bars.`
+				defer
+
+			label follow
+			`You can now clearly see it is a human women, and you indeed know her. She is clearly heading to the exit, by another path than the one you came in by. You know your ship like your pocket and now it will take her a while. She did not notice you after her, yet.`
+			choice
+				`(Wait for her near the exit to stun her).`
+					goto wait
+				`(Order her to stand down).`
+					goto shout
+				`(Shoot).`
+			label shoot
+			`You hit the crew member in the shoulder and he falls. You go remove his weapon, to be sure, and want to ask for an explanation but the only answer you get is "nemo me impune lacessit", before the crew member loses consciousness. Looking eye in the eye, you could see terror. You put him at a nearby hospital, but when you come back later, he is gone, and no one has seen or heard anything.`
```

Comment:
Is this a 'he', 'she', or another pronoun?

Reply:
I wanted to make it a women in one case and a dude in another but I'll just settle with 'they'; I was unfamiliar with that usage before
This mission can happen multiple times

Reply:
Generally "they" for indeterminate (until reason for using or assuming something else) is best, yea.

Even better to stick with it until person being referred to gives their preference, but a lot of people still find that end of things confusing.

Reply:
I made it they and tried to add the intruder; crew member etc to make it clear who we're talking about